dist.ini

```diff
@@ -0,0 +1,60 @@
+; REF: Dist::Zilla https://metacpan.org/pod/Dist::Zilla
+name = Crypt-OpenSSL-X509
+author = Dan Sully
```

Comment:
you can add additional author lines maybe add Jonas if he thinks its warranted.

I have used the following in some of my projects where it made sense

[Meta::Maintainers]
maintainer = 

[Meta::Contributors]
contributor =

Reply:
```
[Meta::Maintainers]
maintainer =

[Meta::Contributors]
contributor =
```

Looks good, I will populate that accordingly.